README.md

```diff
@@ -1,239 +1,99 @@
--   When you install the plug-in from *an online registry or a local package*, the plug-in starts, however, you must update your existing Zowe CLI profiles manually.
-
-## Software requirements
-
-Before you install and use the plug-in:
-
--   Install Zowe CLI on your computer.
-    For more information, see [Installing Zowe CLI](https://docs.zowe.org/stable/user-guide/cli-installcli.html).
--   NPM version 7 (npm@7) and NPM version 8 (npm@8) requirements:
-    On Windows operating systems, ensure that your computer is connected to the Internet and issue the following command ***before*** you install the plug-in:
-    ```
-    npm install -g prebuild-install
-    ```
-### Linux (graphical)
-
--   Install the packages `gnome-keyring` and `libsecret` (or `libsecret-1-0` on Debian and Ubuntu).
-
-### Linux (headless)
-
--   Install the packages listed for graphical Linux above.
-
--   Run the commands below to unlock the Gnome keyring. The keyring must be unlocked again for each new user session.
-
-    The second command will prompt for your password. Press Ctrl+D twice when you have finished typing it.
-    ```bash
-    export $(dbus-launch)
-    gnome-keyring-daemon -r --unlock --components=secrets
-    ```
-
--   To automatically unlock the Gnome keyring at log on:
-
-    **Note:** The following steps have been tested on CentOS, SUSE, and Ubuntu. Results may vary on other Linux distributions.
-
-    1. Install the PAM module for Gnome keyring. The package name depends on your distribution:
-
-        - `gnome-keyring-pam` - CentOS, Fedora, SUSE
-        - `libpam-gnome-keyring` - Debian, Ubuntu
-
-    1. Make the following edits to the files `/etc/pam.d/login` (for TTY login), and `/etc/pam.d/sshd` if it exists (for SSH login).
-
-        - Add this line at the end of the `auth` section:
-            ```
-            auth optional pam_gnome_keyring.so
-            ```
-        - Add this line at the end of the `session` section:
-            ```
-            session optional pam_gnome_keyring.so auto_start
-            ```
-
-    1. Add the following lines to `~/.bashrc`. This will launch DBus which is required by Gnome keyring, and start the keyring daemon so it is ready to be used by Zowe CLI commands.
-
-        ```bash
-        if test -z "$DBUS_SESSION_BUS_ADDRESS" ; then
-          exec dbus-run-session -- $SHELL
-        fi
-
-        gnome-keyring-daemon --start --components=secrets
-        ```
-    
-    1. Reboot your machine and run a Zowe CLI command that uses secure credentials to test that automatic unlock of the keyring works.
-    
-### zLinux
-
-- Follow all installation steps above for Linux
-
-- Install the following packages:
-  - Python 3 (> 3.6.0)
-  - `make`
-  - `gcc-c++`
-  - `libsecret-devel`
+- Zowe CLI on your computer. For more information, see [installing Zowe CLI](https://docs.zowe.org/stable/user-guide/cli-installcli.html).
+- Install [Node.js](https://nodejs.org/en/download) v16.0 or later.
```

Comment:
- I don't think it's necessary to include "on your computer"
- Remove the verb "install" b/c this is a list of items (according to the sentence in Line 15), not actions.